services/meta/association.md

```diff
@@ -3,29 +3,40 @@ This service allows creating a direct association between a source device (e.g.
+-----|----------------------------|------------|----------------|--------------------------------------------------------------------------------------------------------------------------------|
+in   | cmd.association.add        | object     |                | Add members to a specific association group.                                                                                   |
+in   | cmd.association.delete     | object     |                | Remove a single memeber from an association group.                                                                             |
+in   | cmd.association.delete_all | int        |                | Remove all memebers from an association group.                                                                                 |
+in   | cmd.association.get_report | int        |                | Request report of a specific association group.                                                                                |
+out  | evt.association.report     | object     | `max_supports` | Reports members of a specific association. Property `max_supports` defines how many devices can be added to this group.        |
+in   | cmd.groups_info.get_report | null       |                | Z-Wave only. Request detailed information about supported association groups.                                                  |
+out  | evt.groups_info.report     | object     |                | Z-Wave only. Returns a list of supported associations groups with descriptions. See [`association_info_object`](#definitions). |
+
+### Definitions
+
+* `association_info_object` is an object describing a specific association group
+
+| Field        | Example                                   | Description                                                       |
+|--------------|-------------------------------------------|-------------------------------------------------------------------|
+| `group_num`  | `1, 3, 5`                                 | Id of the supported group.                                        |
+| `trigger`    | `"Sensor - Temperature", "Control Key 2"` | General description of the trigger type.                          |
+| `name`       | `"Lifeline", "Dimmer (S1)"`               | Name of the group, specified by the device manufacturer.          |
+| `commands`   | `[{"class": 128, "command": 3}]`          | List of command class and command pairs sent by this association. |
+
+* `trigger` describes the event which will trigger sending an association command. This can be one of the following:
+  * `Lifeline` - special, used only fro receiving reports from the device.
+  * `Control` - triggered by pressing a physical button, eg. `"Control Key 3"`.
+  * `Sensor` - triggered by a sensor report, eg. `"Sensor - Humidity"`.
+  * `Notification` - triggered by an alarm, eg. `"Notification - Smoke"`.
+  * `Meter` - triggered by a meter report, eg. `"Meter - Electric"`.
+   
 #### Examples
 
 _IMPORTANT_
```

Comment:
These should be put in block but most likely not in the Examples section but maybe in the definitions for related objects.

Reply:
Moved to definitions